build-tools/packages/build-cli/src/library/policy/index.ts

```diff
@@ -0,0 +1,11 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { policyHandlers } from "@fluidframework/build-tools";
```

Comment:
These handlers and all the policy-supporting types should be moved from build-tools to build-cli, but I'll do that separately.